src/styles/blocks/details.scss

```diff
@@ -0,0 +1,222 @@
+.details {
+  &__title {
+    margin: 82px 0 48px;
+    padding-top: 10px;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 32px;
+    letter-spacing: -1.5px;
+    text-align: center;
+  }
+
+  &__pictures {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    .first {
+      background: url(../img/content/details/details-first.png);
+    }
+    .second {
+      background: url(../img/content/details/details-second.png) ;
+    }
+    .third {
+      background: url(../img/content/details/details-third.png) ;
+    }
+    .fourth {
+      background: url(../img/content/details/details-fourth.png) ;
+    }
+    .fifth {
+      background: url(../img/content/details/details-fifth.png) ;
+    }
+    .sixth {
+      background: url(../img/content/details/details-sixth.png) ;
+    }
+  }
+
+  &__picture {
+    width: 130px;
+    height: 130px;
+  }
+
+  &__subtitle {
+    margin-bottom: 8px;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 34px;
+  }
+
+  &__description {
+    width: 280px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 22px;
+    margin-bottom: 48px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    font-family: Poppins;
+    background: #fff;
+    border-radius: 40px;
+    padding: 14px 131px;
+    width: 278px;
+    color: $textColorBlack;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 27px;
+  }
+
+  @media (min-width: 744px) {
+    &__title {
+      font-size: 48px;
+      font-weight: 500;
+      line-height: 48px;
+    }
+
+    &__button {
+      max-width: 392px;
+      margin: 0 auto;
+    }
+
+    &__description {
+      width: 600px;
+    }
+
+    &__pictures {
+      display: flex;
+      gap: 24px;
+      justify-content: space-between;
+      margin-bottom: 16px;
+
+      .first {
+        width: $width-big-picture;
+        height: $height-pictures;
```

Comment:
You now have the same `height` for all pictures for this `media` rule, it is better to set it once for `details__picture` and not copy the same thing.